subprojects/execution/src/main/java/org/gradle/internal/execution/history/changes/OutputFileChanges.java

```diff
@@ -16,20 +16,13 @@
 
 package org.gradle.internal.execution.history.changes;
 
-import org.gradle.internal.RelativePathSupplier;
-import org.gradle.internal.fingerprint.FileSystemLocationFingerprint;
-import org.gradle.internal.fingerprint.impl.DefaultFileSystemLocationFingerprint;
-import org.gradle.internal.snapshot.FileSystemLocationSnapshot;
 import org.gradle.internal.snapshot.FileSystemSnapshot;
-import org.gradle.internal.snapshot.RelativePathTracker;
-import org.gradle.internal.snapshot.SnapshotVisitResult;
 
-import java.util.LinkedHashMap;
-import java.util.Map;
 import java.util.SortedMap;
 
 public class OutputFileChanges implements ChangeContainer {
 
+    private static final OutputChangeDetector OUTPUT_CHANGE_DETECTOR = new OutputChangeDetector(NormalizedPathChangeDetector.INSTANCE);
```

Comment:
Why pass the `NormalizedPathChangeDetector` as a parameter here when there's only this one instance of `OutputChangeDetector`?

Reply:
I inlined `OutputChangeDetector`.